src/components/ArhMilitaryRegStatus/ArhMilitaryRegStatus.jsx

```diff
@@ -0,0 +1,22 @@
+import './ArhMilitaryRegStatus.scss';
+
+export default function ArhMilitaryRegStatus() {
```

Comment:
```suggestion
export default function ArchiveMilitaryRegistrationStatus() {
```
Надо писать без сокращений, иначе тяжело будет запомнить где как сокращено